### PR TITLE
Changelog cleanup

### PR DESCRIPTION
**Pull request by ronso0**
* **Music Library** is really large, I've split it into
   * Tracks Table
   * Sidebar
   * Search
   * Track Menu
   * Backend / Database

* Consolidate **Effects**

### CODE REVIEW

**Comment by ronso0** [comment]
Meeeh, AppStream metadata doesn't support h4 (header level 4)??
https://github.com/mixxxdj/mixxx/actions/runs/7698172963/job/20976859344?pr=12686

File: CHANGELOG.md [hunk 1]
```diff
@@ -359,11 +358,11 @@
 * new: Yaeltex MiniMixxx controller mapping [#4350](https://github.com/mixxxdj/mixxx/pull/4350)
 * Behringer DDM4000 mixer: Update controller mapping [#4262](https://github.com/mixxxdj/mixxx/pull/4262) [#4799](https://github.com/mixxxdj/mixxx/pull/4799)
 * Hercules DJ Console RMX: Replace not defined CO name pitch_reset by pitch_set_default [#12441](https://github.com/mixxxdj/mixxx/pull/12441)
-* Korg nanoKONTROL2: Removed Mixco scripts [#2682](https://github.com/mixxxdj/mixxx/pull/2682)
-* Korg nanoKONTROL2: Don't try to configure more than 4 main decks [#12322](https://github.com/mixxxdj/mixxx/pull/12322) [#12317](https://github.com/mixxxdj/mixxx/issues/12317)
 * MAudio Xponent: Removed along with Mixco scripts [#2682](https://github.com/mixxxdj/mixxx/pull/2682)
-* MIDI4lights: Give beginTimer callbacks the anonymous function expression form [#12048](https://github.com/mixxxdj/mixxx/pull/12048)
 * Novation Twitch: Removed along with Mixco scripts [#2682](https://github.com/mixxxdj/mixxx/pull/2682)
+* Korg nanoKONTROL2: Removed along with Mixco scripts [#2682](https://github.com/mixxxdj/mixxx/pull/2682)
```
Comment (daschuer) [inline CHANGELOG.md:363]: I don't really mind, but my original Idea was to keep the order alphabetically.

Comment (ronso0) [reply CHANGELOG.md:363]: Ah, okay, I'll try to restore that.

**Review by daschuer — APPROVED** [review]
Thank you very much!

**Pull request merged**